source/playbooks/playbook-permissions.rst

```diff
@@ -0,0 +1,89 @@
+Roles and Permissions
+=====================
+
+There are different ways to access and interact with playbooks, and access is controlled in the System Console in the form of permissions. Permissions can be granted in a variety of ways, to allow for different combinations of access, visibility, and roles. Two permission schemes are provided in Mattermost:
+
+* **System Scheme:** Applies permissions universally across all teams and channels.
+* **Team Override Schemes:** Allow admins to customize permissions for each team (available in Mattermost Enterprise and Professional).
+
+You can set the default permissions granted to System Admins, Team Admins, Channel Admins, Playbook Admins, Guests (if enabled), and All Members. The permissions granted in the System Scheme apply system-wide, meaning:
+
+* **Guests:** If Guest Accounts are enabled, permissions apply to guest users in all channels, in all teams.
+* **All Members:** Permissions apply to all members, including Admins, in all channels, in all teams.
+* **Channel Administrators:** Permissions apply to all Channel Admins in all channels, in all teams.
+* **Playbook Admins:** Permissions apply to all Playbook Admins in all channels, in all teams.
+* **Team Administrators:** Permissions apply to all Team Admins, in all teams.
+
+For more information about System and Team Override Schemes, refer to the `Advanced Permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__ documentation.
+
+.. note::
+
+  Before changing permissions, ensure that you or another member have Admin access in Playbooks otherwise changing configurations could result in being unable to use Playbooks.
```

Comment:
Might be helpful to show how they can do this: navigate to a playbook -> `Manage Access` menu -> change their role from member to playbook admin

Reply:
Great point, thank you!